src/components/NumberInput/NumberInput.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { Props } from 'react';
+import { TextInput } from 'react-native';
+import styles from './styles';
+
+interface PhoneNumberInputProps {
```

Comment:
nit: seems like this should be renamed to be more general since we're no longer using this for just phone number (not using it for phone number at all)